### PR TITLE
Updates to EN September campaign

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- add links to actions
- add JS to tweet action
- make entire action clickable
- tweaks to the background texture
- theme buttons
- optimize and reorganize images

## Issue / Bugzilla link

#9334 

## Testing

https://www-demo5.allizom.org/en-US/firefox/unfck/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/campaign/unfck/index.html [hunk 1]
```diff
@@ -72,27 +69,34 @@ <h3 class="c-item-title">{{ title }}</h3>
     </ul>
   {% endcall %}
 
-  <h2 class="c-section-title">The Unfcking Checklist</h2>
+  <h2 class="c-section-title">The Checklist</h2>
 
   <div class="mzp-l-content mzp-t-content-lg">
+    {% set tweet_text='2020 might be fcked, but the internet doesn’t have to be. Here are 5 easy ways we can all help turn things around.' %}
+    {% set tw_url = settings.CANONICAL_URL + '/firefox/unfck/' %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/campaign/unfck/index.html:76]: Should the tweet URL be firefox.com/unfck here?

File: media/css/firefox/unfck.scss [hunk 2]
```diff
@@ -197,6 +197,23 @@ main {
...
+            color: $color-ink-90;
+        }
+
+        &:focus {
+            box-shadow: $field-focus-ring-dark;
+            color: $color-ink-90;
+        }
+    }
+
     .fx-privacy-link {
```
Comment (alexgibson) [inline media/css/firefox/unfck.scss:217]: We should really show the privacy link here imho, as it's likely a legal requirement?

File: bedrock/firefox/templates/firefox/campaign/unfck/index.html [hunk 3]
```diff
@@ -21,25 +21,22 @@
...
-    </div>
-  </div>
+    <a id="{{ id }}" class="c-item-unfck" href="{{ link_url }}{% if link_include_utms %}{{ referrals }}{% endif %}" target="_blank" {% if link_include_utms or link_is_external %}rel="external noopener" data-cta-type="link" data-cta-text="Checklist: {{ id }}"{% endif %}>
+      <h3 class="c-item-title">{{ title }}</h3>
+      <div class="c-item-img">
+        <picture class="c-item-gif">
+          {% if src_still %}
+          <source srcset="{{ src_still }}" media="(prefers-reduced-motion: reduce)" loading="lazy" width="280" height="280"></source>
+          {% endif %}
+          <img src="{{ src }}" loading="lazy" width="344" height="344">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/campaign/unfck/index.html:31]: Should the image sizes here match `<source>`?
```suggestion
          <img src="{{ src }}" loading="lazy" width="280" height="280">
```

File: bedrock/firefox/templates/firefox/campaign/unfck/index.html [hunk 4]
```diff
@@ -72,27 +69,34 @@ <h3 class="c-item-title">{{ title }}</h3>
     </ul>
   {% endcall %}
 
-  <h2 class="c-section-title">The Unfcking Checklist</h2>
+  <h2 class="c-section-title">The Checklist</h2>
 
   <div class="mzp-l-content mzp-t-content-lg">
+    {% set tweet_text='2020 might be fcked, but the internet doesn’t have to be. Here are 5 easy ways we can all help turn things around.' %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/campaign/unfck/index.html:75]: Should this now be 6 easy ways?

**Review by alexgibson — APPROVED** [review]
A couple of minor things / questions, but none of them blocking if you would rather tackle them in another PR. r+

File: bedrock/firefox/templates/firefox/campaign/unfck/index.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/campaign/unfck/index.html:76]: I think it should probably be a mz.la shortened URL that has some UTMs built into it but I haven't requested it yet.

File: media/css/firefox/unfck.scss (hunk 2 above)
Comment (stephaniehobson) [reply media/css/firefox/unfck.scss:217]: It's not in the global nav and I remember Jennifer talking to legal and getting the okay to leave it off in other places. Do you think we need to check with someone?

File: bedrock/firefox/templates/firefox/campaign/unfck/index.html (hunk 4 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/campaign/unfck/index.html:75]: I think we're losing one of them and going back to 5. This is something I should double check prior to launch but I think it's okay to leave now.

**Pull request merged**